rdkit/Chem/Pharm2D/UnitTestGenerate.py

```diff
@@ -0,0 +1,30 @@
+# $Id$
+#
+#  Copyright (C) 2002-2006  greg Landrum and Rational Discovery LLC
```

Comment:
I think this is a more general question on how to handle the legal blurb. I understand the situation is more complex due to the history of the code, however what about having a generic statement in each file referring to the license.txt ((C) Rational Discovery LLC, Greg Landrum, and Julie Penzotti) file for all original files (btw. year there is 2006-2015). But then, what to do about modifications?